### PR TITLE
Add warning that Yul is still experimental

### PR DESCRIPTION
**Pull request by axic**
Depends on #6369.

### CODE REVIEW

**Review by chriseth — DISMISSED** [review]

File: libsolidity/interface/StandardCompiler.cpp [hunk 1]
```diff
@@ -886,6 +891,10 @@ Json::Value StandardCompiler::compileYul(InputsAndSettings _inputsAndSettings)
 		return output;
 	}
 
+	// TODO: move this warning to AssemblyStack
+	output["errors"] = Json::arrayValue;
+	output["errors"].append(formatError(true, "Warning", "general", "Yul is still experimental. Please use the output with care."));
```
Comment (axic) [inline libsolidity/interface/StandardCompiler.cpp:896]: Better wording needed here.

**Comment by chriseth** [comment]
#6367 has been merged - please rebase.

**Review by chriseth — DISMISSED** [review]

**Comment by axic** [comment]
Rebased.

File: solc/CommandLineInterface.cpp [hunk 2]
```diff
@@ -874,6 +874,10 @@ bool CommandLineInterface::processInput()
 				endl;
 			return false;
 		}
+		serr() <<
```
Comment (chriseth) [inline solc/CommandLineInterface.cpp:877]: Added this.

**Comment by axic** [comment]
I think this looks ok now. WIll merge.

**Pull request merged**